gpu-service/main.py

```
@@ -73,20 +79,21 @@ def listen_bucket_events(minio_client, bucket_name, q):
 
 try:
-    qdrant.get_collection(collection_name='faces')
-    logger.info("Collection 'faces' already exists.")
-except Exception:
-    logger.info("Collection 'faces' not found, creating it.")
-    qdrant.create_collection(
-        collection_name='faces',
-        vectors_config=models.VectorParams(size=128, distance=models.Distance.COSINE)
-    )
+    ds = deeplake.open(DEEPLAKE_PATH, creds=DEEPLAKE_CREDS)
+    logger.info(f"Loaded existing DeepLake dataset at {DEEPLAKE_PATH}")
```

Ruff (0.14.7)

Logging statement uses f-string

(G004)